### ISSUE TITLE
Is there a way to add "--parallel" when cmake build is done?

### ISSUE THREAD

**Issue by EricBackus** [pfultz2/cget#73]
Newer versions of CMake support a "--parallel" command-line option. I'd like to try using it from cget, but there doesn't appear to be a way for cget to send arbitrary strings to the CMake command-line. Could something like this be added?

**Comment by pfultz2** [comment]
It looks like you could set the [`CMAKE_BUILD_PARALLEL_LEVEL.`](https://cmake.org/cmake/help/latest/envvar/CMAKE_BUILD_PARALLEL_LEVEL.html) env variable. 

It might be good to extend cget to support this option.

**Comment by EricBackus** [comment]
You're right, the CMAKE_BUILD_PARALLEL_LEVEL env variable will probably meet this immediate need. 

However, I think a generic way to add to the CMake command-line would still be very useful. For example, with my own packages, I would probably want to add --warn-uninitialized to help find possible problems in my CMakeLists.txt. Or, when building with MSVC, I might want to add "-- /verbosity:minimal" to cut down on the extremely verbose Visual Studio output. It would be best to have separate control over CMake generation vs. CMake build.
